public/docs/TermsAndConditionsUk.html

```diff
@@ -12,9 +12,8 @@
         <h1 class="font-sans flex-1 md:text-xl font-extrabold uppercase">
             ПУБЛІЧНИЙ ДОГОВІР (ОФЕРТА) ЩОДО НАДАННЯ ДОСТУПУ ДО СЕРВІСУ ТА ПОСЛУГ 
         </h1>
-        <a class="button flex-4 w-24 mr-1 pb-2 border text-white dark:border-dark-5" href="https://dataocean.us/" target="_blank">
-            <ion-icon class="text-white" name="arrow-back-outline"></ion-icon>
-            Назад
+        <a class="flex-4 ml-2 cursor-pointer border-0" href="https://dataocean.us/" target="_blank" style="border-bottom:0px">
+            <img src="../images/logo-white.svg" alt="LOGO">
```

Comment:
maybe?
```suggestion
            <img src="/images/logo-white.svg" alt="LOGO">
```

Reply:
no, does'nt work as you propose. And i dont understand why (
I am looking for this solution, maybe 1 hour..

Reply:
![image](https://user-images.githubusercontent.com/38906004/111112417-cdc0f500-8568-11eb-885a-e61a7e06c32e.png)
![image](https://user-images.githubusercontent.com/38906004/111112437-d74a5d00-8568-11eb-8532-e3a338c5d9e8.png)